roboflow/util/prediction.py

```diff
@@ -179,6 +197,8 @@ def save(self, output_path="predictions.jpg", stroke=2, transparency=60):
             )
 
         elif self["prediction_type"] == CLASSIFICATION_MODEL:
+            if self["top"] in self.colors.keys():
```

Comment:
what's top?

Reply:
"top" is the predicted classification class based on our schema there